packages/unchained-client/src/parser/thorchain.ts

```diff
@@ -112,7 +112,7 @@ export interface ParserArgs {
   midgardUrl: string
 }
 
-const streamingSwapRegex = /:\d+\/\d+\/\d+:/
+const streamingSwapRegex = /:(\d+|(\d+(?:e[+-]?\d+)?))\/\d+\/\d+:/
```

Comment:
update supports scientific notation number in the first streaming swap parameter (which I found as valid in testing)

Reply:
🤯

Reply:
thanks chatgpt!